src/components/Lottie.js

```diff
@@ -30,10 +30,9 @@ const LottieAnimation = ({
       options={lottieOptions}
       height="100%"
       width="100%"
-      ariaRole={null}
       ariaLabel={ariaLabel}
       tabIndex={-1}
-      className={hideInPercy && 'hide-in-percy'}
+      className={hideInPercy ? 'hide-in-percy' : undefined}
```

Comment:
Is your className value `undefined` ? How does it appear in the final dom element?

Reply:
I had the following console error: 
`Warning: Received `false` for a non-boolean attribute `className`.

If you want to write it to the DOM, pass a string instead: className="false" or className={value.toString()}.

If you used to conditionally omit it with className={condition && value}, pass className={condition ? value : undefined} instead.`

So I just applied their suggestion in the last sentence :)